src/scripts/destinyTrackerApi/bulkFetcher.js

```diff
@@ -0,0 +1,81 @@
+import { itemListBuilder } from './itemListBuilder.js';
+
+class bulkFetcher {
```

Comment:
We haven't started using classes a lot yet, but in general the convention is that classes start with a capital letter.